crypto/keyring/keyring_test.go

```diff
@@ -97,12 +101,12 @@ func TestKeyManagementKeyRing(t *testing.T) {
 	addr, err = sdk.AccAddressFromBech32("cosmos1yq8lgssgxlx9smjhes6ryjasmqmd3ts2559g0t")
 	require.NoError(t, err)
 	_, err = kb.KeyByAddress(addr)
-	require.NotNil(t, err)
+	require.Error(t, err)
 
 	// list shows them in order
 	keyS, err := kb.List()
 	require.NoError(t, err)
```

Comment:
I get the following error `unexpected EOF` while invoking [ProtoUnmarshalRecord](https://github.com/cosmos/cosmos-sdk/blob/kerying-migration/crypto/keyring/keyring.go#L500). This is full test [output](https://play.golang.org/p/8AP8WEMJcgV). In this output I get this error at line 16 and 18 .Can you advise please how to fix it?